Recipe-App/Recipe-App/Models/Recipe.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Recipe.swift
+//  Recipe-App
+//
+//  Created by Gontse Ranoto on 2021/06/11.
+//
+
+import Foundation
+
+struct Recipe {
```

Comment:
Can you extend Recipe with Identifiable and Codable

Reply:
Identifiable does not conform to codable...

Reply:
Okay that's weird.

Reply:
Okay re-request review so that I can approve.